region/lowernorfair/east/Pillar Room.json

```diff
@@ -193,79 +204,154 @@
+              {"acidFrames": 30}
             ]}
           ]},
           {"and": [
-            {"heatFrames": 750},
-            {"acidFrames": 36}
+            "Gravity",
+            {"heatFrames": 660},
+            {"acidFrames": 30}
+          ]},
+          {"and": [
+            "canTrickyJump",
+            "canSuitlessLavaDive",
+            {"heatFrames": 720},
+            {"acidFrames": 52}
+          ]}
+        ]},
+        {"or": [
+          "canTrickyJump",
+          {"and": [
+            "Gravity",
+            {"heatFrames": 180},
+            {"acidFrames": 56}
+          ]},
+          {"and": [
+            "canSuitlessLavaDive",
+            {"heatFrames": 240},
+            {"acidFrames": 96}
           ]}
         ]}
       ],
-      "reusableRoomwideNotable": "Lower Norfair Pillar Room with Power Bombs",
+      "reusableRoomwideNotable": "Lower Norfair Pillar Room with Two Power Bombs",
       "note": [
         "Place the PBs next to the pillars in order to only use 2.",
         "Minimize acid by unmorphing high to land back on the jump spot or walljumping before placing the bomb."
       ]
     },
     {
       "link": [1, 2],
-      "name": "Pillar Room with Three Power Bombs (Left to Right)",
+      "name": "Three Power Bombs",
       "notable": true,
       "requires": [
-        "h_canNavigateHeatRooms",
+        "canCarefulJump",
         "Morph",
         {"ammo": {"type": "PowerBomb", "count": 3}},
-        {"heatFrames": 670}
+        {"or": [
+          "canTrickyJump",
+          {"enemyDamage": {
+            "enemy": "Puromi",
+            "type": "contact",
+            "hits": 1
+          }}
+        ]},
+        {"heatFrames": 670},
+        {"or": [
+          "canTrickyJump",
+          {"and": [
+            "Gravity",
+            {"heatFrames": 180},
+            {"acidFrames": 56}
+          ]},
+          {"and": [
+            "canSuitlessLavaDive",
+            {"heatFrames": 240},
+            {"acidFrames": 96}
+          ]}
+        ]}
       ],
-      "reusableRoomwideNotable": "Lower Norfair Pillar Room with Power Bombs",
       "note": [
         "The power bombs can be placed pretty far from the next pillar in line.",
-        "PB1 - Above the mound of dirt on the ground.  PB2 - On pillar 2.  PB3 - Near pillar 5."
+        "PB1 - Above the mound of dirt on the ground.  PB2 - On pillar 2 (not on the Puromi Fire Snake).  PB3 - Near pillar 5."
+      ]
+    },
+    {
+      "link": [1, 2],
+      "name": "Lower Norfair Pillar Room with Bombs (Left to Right)",
+      "notable": true,
+      "entranceCondition": {
+        "comeInRunning": {
+          "minTiles": 1,
+          "speedBooster": "any"
+        }
+      },
+      "requires": [
+        "h_canUseMorphBombs",
+        "canWallJumpInstantMorph",
+        "canTrickyJump",
+        "canResetFallSpeed",
+        "canUnmorphBombBoost",
+        "canSuitlessLavaDive",
+        {"heatFrames": 1320},
+        {"acidFrames": 128}
```

Comment:
Would this be reduced much with gravity? (and in the reverse direction)
I think it would still require suitless lava dive with this many acid frames

Reply:
The way Bob does it, Gravity wouldn't help much.
But without Gravity, the strat instantly fails if you fall behind, so there could be a version where you play a little worse and take twice as much damage.